protocol/src/main/java/org/fido/iot/protocol/CryptoService.java

```
@@ -99,7 +99,11 @@ public CryptoService(String[] algNames) {
     }
     // no algorithm found in priority list
     if (rnd == null) {
-      rnd = new SecureRandom();
+      try {
+        rnd = SecureRandom.getInstanceStrong();
+      } catch (NoSuchAlgorithmException e) {
+        e.getMessage();
```

Is this intentionally ignoring the exception?